index.html

```diff
@@ -1,27 +1,42 @@
+<!DOCTYPE html>
 
-<html>
+<html lang="en">
 <head>
-	<meta charset="utf-8"><link rel="stylesheet" href="/main.css">	<title>Alia Borroho</title>
-	</head>
-	<body>
-	<p>
-<img src="https://mail.google.com/mail/u/0/?ui=2&ik=326bbbf37a&view=fimg&th=14e433814216e904&attid=0.1&disp=inline&realattid=f_ibiz1aqf0&safe=1&attbid=ANGjdJ_4IwoHJMyjD0kY0nUMODfgzmagalIKQQd9a-V4W0qpglNT-qXetoLntsvpLgUqD1PIoMZL7uJk6iLe216XqidbUWUj5cT8QQwcThIoxErnHzH_2m0hmG6oSEw&ats=1435646996109&rm=14e433814216e904&zw&sz=w1252-h562">
-      </p>
-	<h1>Alia Borroho IY 2015</h1>
-<h2 style:"text-align:center">aliaborroho@gmail.com</h2>
+    <meta charset="utf-8">
 
+    <title>Alia Borroho</title>
 
-<p style="text-align:center;font-family:Verdana">
-  I have always wanted to understand the inner-workings of the 'internet of things'. I really like the thought of learning different concepts, languages, commands. I can try to put it in a metaphor. Being a developer is a digital version of how normal human beings view city development. It's like building a city. You have to build a foundation first and build upon it, but first you have to make streets and avenues and plan the sewage pipes all runnng underground. One thing depends of the fate of the next. I believe being a developer will be fascinating, it already is.</p>
-<p style="font-family:Verdana">Social Media Links</p>
-<p>
-	<ul>
-		<li style="font-size:12px;font-family:Verdana">https://www.linkedin.com/in/aliaborroho</li>
-		<li style="font-size:12px;font-family:Verdana">https://twitter.com/AliaBorroho1</li>
-		<li style="font-size:12px;font-family:Verdana">https://www.facebook.com/AliaBorroho</li>
-		</ul>
-</p>
-</body>
+</head>
+<body>
+<span>
+     <a href="http://tinypic.com?ref=eg9zeb" target="_blank"><img src="http://i60.tinypic.com/eg9zeb.jpg" border="0" alt="alia-pic"></a>
```

Comment:
The URL `http://tinypic.com?ref=eg9zeb` is not a valid url. Just have an image tag here.